### ISSUE TITLE
Sierra issue with BaseSystemBinaries.pkg (in GM)

### ISSUE THREAD

**Issue by jmichaelbarker** [ntkme/iesd#7]
The `BaseSystemBinaries.pkg` package has been renamed to `BaseSystemResources.pkg`, resulting in `iesd` failing to complete.

Edit: This issue also occurs with the latest version of El Capitan - `Could not open package for expansion: /var/folders/d5/[...]/System/Installation/Packages/BaseSystemBinaries.pkg
Failed during: /usr/bin/env pkgutil --expand /var/folders/d5/[...]/System/Installation/Packages/BaseSystemBinaries.pkg /var/folders/d5/[...]/BaseSystemBinaries.pkg`

**Comment by ntkme** [comment]
There are quite a few issues with El Capitan & later:
- Package name and content changes
- Package format (pbzx stream)
- GUID Partition Map on BaseSystem.dmg

I don't have time working on these issues lately. Pull requests are very welcome.  
Also, as I mentioned on my blog, upgrade Yosemite VM to a newer version should work.

**Comment by jmichaelbarker** [comment]
Yep, noticed those other issues a little after creating this issue. If I get the chance I'll attempt to fix some of these and make a pull request.
